cmd/promu.go

```diff
@@ -165,3 +165,12 @@ func stringInSlice(needle string, haystack []string) bool {
 	}
 	return false
 }
+
+func hasRequiredConfigurations(configVars []string) (bool, error) {
```

Comment:
Ok. It's more common to leave that to the caller. `fatal(hasRequiredConfigurations(...))` adds a mental step.